src/applications/vaos/components/ConfirmationDirectScheduleInfo.jsx

```diff
@@ -1,15 +1,28 @@
 import React from 'react';
-import moment from 'moment';
+import moment from '../utils/moment-tz.js';
 import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
 import FacilityAddress from './FacilityAddress';
+import AddToCalendar from './AddToCalendar';
+import { getFacilityAddress } from '../utils/appointment';
+import {
+  getTimezoneAbbrBySystemId,
+  getTimezoneBySystemId,
+} from './../utils/timezone';
 import { PURPOSE_TEXT } from '../utils/constants';
 
 export default function ConfirmationDirectScheduleInfo({
   data,
   facilityDetails,
   clinic,
   pageTitle,
+  appointmentLength,
 }) {
+  const dateTime = data.calendarData.selectedDates[0].datetime;
+  const timezone = getTimezoneBySystemId(data.vaSystem);
+  const momentDate = timezone
+    ? moment(dateTime).tz(timezone.timezone, true)
```

Comment:
`.tz()` applies the string formatted timezone (e.g. `"Americas/Chicago"`) to the moment date object, and passing `true` as the second parameter keeps the time as is, without adjusting it.  Without `true`, it would assume the supplied date is in local time zone and then adjust the hours it to the supplied timezone.